lib/jsdom/living/navigator/MimeType-impl.js

```diff
@@ -0,0 +1,3 @@
+"use strict";
+
+exports.implementation = class MimeType {};
```

Comment:
The reason this works is because it's not actually possible to get an instance of one of these, I guess? Fascinating.